### PR TITLE
VSCode BUILD.MD update for production build.

### PR DESCRIPTION
**Pull request by MartinBalin**
Correction of how to build for release etc. with prerequisities.

### CODE REVIEW

**Review by dbalek — APPROVED** [review]
Looks fine.

File: java/java.lsp.server/vscode/BUILD.md [hunk 1]
```diff
@@ -35,10 +42,9 @@ To build the VS Code extension invoke:
 ```bash
 netbeans$ ant build
 netbeans$ cd java/java.lsp.server
-java.lsp.server$ ant build-vscode-ext -D3rdparty.modules=.*nbjavac.*
+java.lsp.server$ ant build-vscode-ext -Dvsix.version=x.y.z
 ```
-
-The `3rdparty.modules` property doesn't have to be set at all.
```
Comment (jtulach) [inline java/java.lsp.server/vscode/BUILD.md:41]: Either the `3rdparty.modules` property remain to be documented, or it shall be removed. If it doesn't work with `m2://` protocol (which it didn't), that should be fixed.

**Comment by JaroslavTulach** [comment]
#2777 makes the `-D3rdparty.modules=.*nbjavac.*` work again. We don't have to promote it as the default option, but I'd suggest to still mention it in the readme. It is the way to make the extension work on JDK8.

**Comment by MartinBalin** [comment]
Options better documented now in 2nd commit.

**Comment by jtulach** [comment]
OK. Just squash the commits into one.

**Pull request closed without merging**